.github/ISSUE_TEMPLATE/improve-the-site.yml

```
@@ -0,0 +1,55 @@
+name: Suggest ideas and updates for the Docs website
+about: Make a suggestions or report a problem on the docs.github.com website.
+labels: engineering
+inputs:
+  - type: description
+    attributes:
+      value: |
+        HUBBERS BEWARE! THE GITHUB/DOCS REPO IS PUBLIC TO THE ENTIRE INTERNET. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.
```

This whole improve the site part is for engineering right? If so, then we can go ahead and do this right?

```suggestion
        HUBBERS BEWARE! Please confirm you're on the github/docs-internal repo and not the github/docs repo before proceeding, unless you want to make your work public to the entire internet. The github/docs repo is open source and public to the entire internet.
```